lib/plugins/rotate_asg_instances_lib/system.rb

```diff
@@ -0,0 +1,80 @@
+class System
```

Comment:
Can you check if https://github.com/acquia/moonshot/blob/master/lib/moonshot/ssh_fork_executor.rb can be used instead of adding this new class?

Reply:
I think this class has way more to provide than the `ssh_fork_executor` and the implementation is also different in `ssh_fork_executor`. We can add this class 💭

Reply:
@AbhishekRana95 It doesn't look like we are using any of the additional functionality that is present in the class you added. We should just use the method already in moonshot if that will do what we need.

Reply:
@kaushik used `ssh_fork_executor` for exec functionality.